src/monattack.cpp

```diff
@@ -514,7 +514,7 @@ bool mattack::acid_accurate(monster *z)
     proj.proj_effects.insert( "NO_DAMAGE_SCALING" );
     proj.impact.add_damage( DT_ACID, rng( 3, 5 ) );
     // Make it arbitrarily less accurate at close ranges
-    z->projectile_attack( proj, target->pos(), rng( 0, 8000 / range ) );
+    z->projectile_attack( proj, target->pos(), 8000 / range );
```

Comment:
You might want to review this forum post about this not being updated in the aiming refactor and therefore being to inaccurate. Consider replacement with a range independent constant.